ldclient/streaming.py

```diff
@@ -46,27 +46,28 @@ def initialized(self):
 
     @staticmethod
     def process_message(store, requester, msg, ready):
-        payload = json.loads(msg.data)
```

Comment:
The indirect message payloads are not json, but instead are a list of flag keys (or one flag key in the case of indirect/patch). We were failing here when attempting to parse the json.